src/test/unit/rx_spi_expresslrs_telemetry_unittest.cc

```diff
@@ -84,30 +84,29 @@ TEST(RxSpiExpressLrsTelemetryUnitTest, TestInit)
 
 static void testSetDataToTransmit(uint8_t payloadSize, uint8_t *payload)
 {
-    uint8_t *data;
-    uint8_t maxLength;
-    uint8_t packageIndex;
+    uint8_t data[ELRS_TELEMETRY_BYTES_PER_CALL] = {0};
```

Comment:
Add an expect statement checking `payload`?

Reply:
This function checks whether the `payload` is split into chunks properly. The `payload` contains sensor specific in this case, I am checking for the `payload` content in the tests themselves, i.e. for GPS data the checks are here:
https://github.com/phobos-/betaflight/blob/elrs-3/src/test/unit/rx_spi_expresslrs_telemetry_unittest.cc#L132

Reply:
OK